sdk/src/transaction.rs

```diff
@@ -36,26 +34,18 @@ impl Instruction {
 /// An atomic transaction
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct Transaction {
-    /// A set of digital signature of `account_keys`, `program_ids`, `last_id`, `fee` and `instructions`, signed by the first
+    /// A set of digital signatures of `account_keys`, `program_keys`, `last_id`, `fee` and `instructions`, signed by the first
```

Comment:
`program_ids` was right, no?

Reply:
@garious Yup, was toying with getting rid of the `id` nomenclature and just calling them `program_keys` for consistency.  Fixed